delfin/db/api.py

```diff
@@ -228,6 +228,64 @@ def storage_pool_delete_by_storage(context, storage_id):
     return IMPL.storage_pool_delete_by_storage(context, storage_id)
 
 
+def controllers_create(context, values):
+    """Create multiple controllers."""
+    return IMPL.controllers_create(context, values)
+
+
+def controllers_update(context, values):
+    """Update multiple controllers."""
+    return IMPL.controllers_update(context, values)
+
+
+def controllers_delete(context, values):
+    """Delete multiple controllers."""
+    return IMPL.controllers_delete(context, values)
+
+
+def controller_create(context, values):
+    """Create a controller from the values dictionary."""
+    return IMPL.controller_create(context, values)
+
+
+def controller_update(context, controller_id, values):
```

Comment:
When would the controller be updated? Should we add this to resources.py in task_manager?